lib/array.rb

```diff
@@ -0,0 +1,9 @@
+class Array
+    def inject_1 (n=1)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis.